src/sentry/api/endpoints/project_details.py

```diff
@@ -516,7 +517,7 @@ def put(self, request: Request, project) -> Response:
         Update various attributes and configurable settings for the given project.
 
         Note that solely having the **`project:read`** scope restricts updatable settings to
-        `isBookmarked` only.
+        `isBookmarked` and `isSubscribed`.
```

Comment:
I'm not actually sure what `isSubscribed` does 😅
It might be ok to just leave it off

Reply:
oh nvr mind I see you figured out what it does lol